src/test/java/io/strimzi/test/container/StrimziKafkaContainerIT.java

```diff
@@ -125,8 +125,10 @@ void testStartContainerWithServerProperties() {
     void testStartContainerWithStrimziKafkaImage() {
         assumeDocker();
 
+        // explicitly set strimzi.custom.image
+        System.setProperty("strimzi.custom.image", "quay.io/strimzi/kafka:latest-kafka-3.0.0");
```

Comment:
Should this be `quay.io/strimzi-test-container/test-container:latest-kafka-3.0.0`? Or if it is just a dummy value to see it is passed properly, it should be something dummy to make it clear what value does it have.

Reply:
In this test, we actually spin up Kafka container so it's needed :).

Reply:
So, how do you know it is/will be compatible? Should you at least use a release version which will not change anymore?

Reply:
> Should you at least use a release version that will not change anymore?

Will do :)